src/components/AddressSearch/index.js

```diff
@@ -254,6 +254,17 @@ function AddressSearch(props) {
             values.state = stateFallback;
         }
 
+        // Some edge-case addresses may lack both street_number and route in the API response, resulting in an empty "values.street"
+        // We are setting up a fallback to ensure "values.street" is populated with a relevant value
+        if (!values.street && details.adr_address) {
+            const streetAddressRegex = /<span\s+class\s*=\s*"street-address"[^>]*>([^<]*)<\/span>/;
+            const adr_address = details.adr_address.match(streetAddressRegex);
+            const streetAddressFallback = _.get(adr_address, [1], null);
```

Comment:
Any reason why we can't use `lodashGet` here instead?

Reply:
I'm already using _.get, is this an old comment I didn't see before?

Reply:
I'm talking about `lodashGet` e.g. here

https://github.com/Expensify/App/blob/b81148810c83bba7de2dcdd22dc4ac9878b0b9c5/src/components/AddressSearch/index.js#L240

Reply:
It's not a big deal since we're all using underscores in this file, but it's nicer to use it when we can!

Reply:
In that case then no there is no specific reason I used _, I pretty much did only because it's already imported and used all over the component so it doesn't really make a difference from what I know.

Reply:
Let's switch it to `lodashGet` as we prefer it in general. Makes it easier if we want to get rid of underscores altogether in future!

Reply:
Done!